### PR TITLE
Add awesome-lint

### PR DESCRIPTION
**Pull request by zckrs**
Hi,

Just add https://github.com/sindresorhus/awesome-lint to a travis build.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -883,7 +883,7 @@
 - [Reddit](https://www.reddit.com/r/node)
 - [Twitter](https://twitter.com/nodejs)
 
-### Miscellaneous
+### Various
```
Comment (sindresorhus) [inline readme.md]: ?

Comment (zckrs) [inline readme.md]: Because the rule no-duplicate-headings. 😥

Comment (sindresorhus) [inline readme.md]: Makes no sense for the rule to enforce that on headings in different sections.

I've disabled the rule for now. (Next release).

Just revert this change.

Comment (zckrs) [inline readme.md]: Done

**Comment by SamVerschueren** [comment]
Should we add the travis badge at the top as well?

**Comment by sindresorhus** [comment]
Nah

**Comment by sindresorhus** [comment]
I appreciate the PR, but `awesome-lint` is not mature enough to be used here. I'll reopen this when it is.

**Comment by zckrs** [comment]
Np 🦄
